packages/back-end/src/routers/organizations/organizations.controller.ts

```diff
@@ -109,6 +110,7 @@ export async function getDefinitions(req: AuthRequest, res: Response) {
     tags,
     savedGroups,
     projects,
+    informationSchemas,
```

Comment:
Do we need this anymore if we removed information schemas from the definitions context?